src/main/java/creativei/manager/impl/LocalityMangerImpl.java

```diff
@@ -29,7 +30,7 @@ public ResponseObject getPincodes(String pincode) {
 
     @Override
     public ResponseObject getLocalityDataByPincode(String pincode) {
-        CityVo city=new CityVo();
+        CityVo city = new CityVo();
```

Comment:
Move city specific initialization code to CityVo and add constructor there. This way you can remove all the setters being called from this method.
CityVo(City city){

}

Reply:
ok. Issue resolved in commit 4ab6df7